config/flags.go

```diff
@@ -388,6 +388,8 @@ func addNodeFlags(fs *pflag.FlagSet) {
 	fs.Float64(TracingSampleRateKey, 0.1, "The fraction of traces to sample. If >= 1, always sample. If <= 0, never sample")
 	fs.StringToString(TracingHeadersKey, map[string]string{}, "The headers to provide the trace indexer")
 	// TODO add flag to take in headers to send from exporter
+
+	fs.String(RuntimeStatePathKey, "", "The path to write runtime state to (including uri, bootstrap address and pid). If empty, runtime state will not be written.")
```

Comment:
IIUC, we leave users total freedom to pick any path it wishes to store this information. I wonder if instead we should enfoce a specific location to look at, somewhere in data dir. I think it's best to provide a standard location to look at

Reply:
I'd go for boolean to enable but fixed path. We'll see if other reviewers have objections to the reduced flexibility. I can't find any as of now.